nano/qt_test/qt.cpp

```diff
@@ -60,6 +62,38 @@ TEST (wallet, status)
 	ASSERT_TRUE (wallet_has (nano_qt::status_types::disconnected));
 }
 
+// this test is modelled on wallet.status but it introduces another node on the network
+TEST (wallet, status_with_peer)
+{
+	nano_qt::eventloop_processor processor;
+	nano::system system (2);
+	auto wallet_l (system.nodes[0]->wallets.create (nano::random_wallet_id ()));
+	nano::keypair key;
+	wallet_l->insert_adhoc (key.prv);
+	auto wallet (std::make_shared<nano_qt::wallet> (*test_application, processor, *system.nodes[0], wallet_l, key.pub));
+	wallet->start ();
+	auto wallet_has = [wallet] (nano_qt::status_types status_ty) {
+		return wallet->active_status.active.find (status_ty) != wallet->active_status.active.end ();
+	};
+	system.nodes[0]->network.udp_channels.insert (nano::endpoint (boost::asio::ip::address_v6::loopback (), 10000), 0);
```

Comment:
Let's use a TCP variant since we want to remove the UDP code soon.

Reply:
Removed the line alltogether since it turned out it is not necessary.
 	system.nodes[0]->network.udp_channels.insert